### PR TITLE
fix: add `VMware.PowerCLI` module to `requiredModules`

### LINKED ISSUE #169: Add `VMware.PowerCLI` module name to `requiredModules`
### Code of Conduct

- [X] I have read and agree to the [Code of Conduct](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-password-management/blob/main/CODE_OF_CONDUCT.md).
- [X] Vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue initial description to help the maintainers prioritize.
- [X] Do not leave "+1" or other comments that do not add relevant information or questions.
- [X] If you are interested in working on this issue or have submitted a pull request, please leave a comment.

### VMware Cloud Foundation

5.1

### PowerShell Version

Windows Powershell 7.3.4

### PowerCLI Version

13.1

### Module Version

1.7.1

### PowerValidatedSolutions Version

2.9.0

### PowerVCF Version

2.4.1

### Guest Operating System

Windows Server 2019

### Environment Details

NA

### Description

The manifest file [`VMware.CloudFoundation.PasswordManagement.psd1`] holds the list of required modules for the functioning of password management module. It lists out "PowerVCF", "PowerValidatatedSolutions", "VMware.vsphere.ssoadmin", but there is one more module to be listed, that is "VMware.PowerCLI". This was caught during CD/CI run where all modules listed in the manifest file will be automatically loaded. Since this was not loaded, one of the powercli commandlet failed and test execution didn't continue.

### Error or Debug Output

`Set-PowerCLIConfiguration` is not a valid cmdlet

### Expected Behavior

Set-PowerCLIConfiguration should be executed successfully

### Actual Behavior

`Set-PowerCLIConfiguration` is not a valid cmdlet

### Steps to Reproduce

Run PPM.Tests.ps1 file which is a pester file under tests folder without installing `VMware.PowerCLI` module

### Log Fragments and Files

_No response_

### Screenshots

_No response_

### References

_No response_

### PR DESCRIPTION
**Pull request by sowjuec**
### Summary

`VMware.PowerCLI` module name was not listed in the required modules list of the manifest file. Added the same.

### Type

- [X] Bugfix
- [ ] Enhancement or Feature
- [ ] Code Style or Formatting
- [ ] Documentation
- [ ] Refactoring
- [ ] Chore
- [ ] Other
        Please describe:

### Breaking Changes?

- [ ] Yes, there are breaking changes.
- [X] No, there are no breaking changes.

### Test and Documentation

- [X] Tests have been completed.
- [ ] Documentation has been added or updated.

### Issue References

   Closes #169

### CODE REVIEW

**Review by tenthirtyam — CHANGES_REQUESTED** [review]
- need to bump the manifest build number
- need to modify the release action to install powercli from psgalley in order to successfully test during the release flow

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -188,10 +188,17 @@ jobs:
               Set-Location '/home/runner/.local/share/powershell/Modules/' 
               foreach ($module in $requiredModules) {
                   $requiredModuleName = $module.ModuleName
-                  New-Item $requiredModuleName -ItemType Directory
-                  Write-Output "INFO: Performing workaround for github.com/PowerShell/PowerShell/issues/7722."
-                  Write-Output "INFO: Creating placeholder manifest for $requiredModuleName at $((Get-Location).Path)/$requiredModuleName/$requiredModuleName.psd1"
-                  New-Item "./$requiredModuleName/$requiredModuleName.psd1" -ItemType File
+                  if ($requiredModuleName -eq 'VMware.PowerCLI') {
+                    Write-Output "INFO: Ignoring installed version of VMware.PowerCLI."
```
Comment (tenthirtyam) [inline .github/workflows/release.yml]:
```suggestion
```

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -21,6 +21,9 @@ jobs:
           Set-PSRepository psgallery -InstallationPolicy trusted
           Write-Output "INFO: Installing module 'VMware.vSphere.SsoAdmin' from PSGallery."
           Install-Module -Name VMware.vSphere.SsoAdmin -confirm:$false -Force
+          Write-Output "INFO: Installing module 'VMware.PowerCLI' from PSGallery."
+          Uninstall-Module -Name VMware.PowerCLI -AllVersions -Force
```
Comment (tenthirtyam) [inline .github/workflows/tests.yml]:
```suggestion
```

Prior versions would not be present on the runner so this can be skipped.

**Review by tenthirtyam — CHANGES_REQUESTED** [review]
Minor changes requested.

**Comment by sowjuec** [comment]
> Minor changes requested.

Taken Care!!!!

**Review by tenthirtyam — APPROVED** [review]
LGTM!

**Pull request merged**